### PR TITLE
[pt] 3.0 Migrations translation

### PR DESCRIPTION
**Pull request by angelitomg**
Start Migrations translate.

### CODE REVIEW

File: pt/migrations.rst [hunk 1]
```diff
@@ -1,13 +1,820 @@
...
+
+    $ composer require cakephp/migrations "@stable"
+
+Para usar o plugin você precisa carregá-lo no arquivo **config/bootstrap.php** da sua aplicação. 
+Você pode usar o :ref:`CakePHP's Plugin shell <plugin-shell>` para carregar e descarregar
+plugins do seu arquivo **config/bootstrap.php**::
+
+    $ bin/cake plugin load Migrations
+
+Ou você pode carregar o plugin editando seu arquivo **config/bootstrap.php** e adicionando a linha:
```
Comment (markstory) [inline pt/migrations.rst]: This needs to end in `::`

File: pt/migrations.rst [hunk 2]
```diff
@@ -1,13 +1,820 @@
...
+
+Visão Geral
+========
+
+Uma migração é basicamente um único arquivo PHP que descreve as mudanças a 
+serem feitas no banco de dados. Um arquivo de migração pode criar ou excluir 
+tabelas, adicionar ou remover colunas, criar índices e até mesmo inserir 
+dados em seu banco de dados.
+
+Aqui um exemplo de migração:
```
Comment (markstory) [inline pt/migrations.rst]: This needs to end in `::`

File: pt/migrations.rst [hunk 3]
```diff
@@ -1,13 +1,820 @@
...
+
+    Plugin::load('Migrations');
+
+
+Adicionalmente, você irá precisar configurar o banco de dados padrão da sua aplicação, 
+no arquivo **config/app.php** como explicado em :ref:`Database Configuration section 
+<database-configuration>`.
+
+Visão Geral
+========
```
Comment (markstory) [inline pt/migrations.rst]: Underlines have to be as long as the text they underline.

File: pt/migrations.rst [hunk 4]
```diff
@@ -1,13 +1,820 @@
...
-    `Github <https://github.com/cakephp/docs>`_ ou use o botão
-    **Improve This Doc** para propor suas mudanças diretamente.
+    Note que este arquivo descreve como o banco de dados deve ser **após** a
+    aplicação da migração. Neste ponto, a tabela ``products``ainda não existe 
+    no banco de dados, nós apenas criamos um arquivo que é capaz de criar a 
+    tabela ``products`` com seus devidos campos ou excluir a tabela caso uma 
+    operação rollback seja executada.
+
+Com o arquivo criado na pasta **config/MIgrations**, você será capaz de executar 
+o comando abaixo para executar as migrações no seu banco de dados:
```
Comment (markstory) [inline pt/migrations.rst]: This needs to end in `::`

File: pt/migrations.rst [hunk 5]
```diff
@@ -1,13 +1,820 @@
...
+    no banco de dados, nós apenas criamos um arquivo que é capaz de criar a 
+    tabela ``products`` com seus devidos campos ou excluir a tabela caso uma 
+    operação rollback seja executada.
+
+Com o arquivo criado na pasta **config/MIgrations**, você será capaz de executar 
+o comando abaixo para executar as migrações no seu banco de dados: 
+
+    bin/cake migrations migrate
+
+O comando seguinte irá executar um rollback na migração e irá excluir a tabela recém criada:
```
Comment (markstory) [inline pt/migrations.rst]: This needs to end in `::`

File: pt/migrations.rst [hunk 6]
```diff
@@ -1,13 +1,820 @@
...
+
+    bin/cake migrations rollback
+
+Criando Migrations
+===================
+
+Arquivos de migração são armazeados no diretório **config/Migrations** da 
+sua aplicação. O nome dos arquivos de migração têm como prefixo a data 
+em que foram criados, no formato **YYYYMMDDHHMMSS_MigrationName.php**. Aqui são exemplos de arquivos de migração:
```
Comment (markstory) [inline pt/migrations.rst]: This needs to end in `::`

File: pt/migrations.rst [hunk 7]
```diff
@@ -129,36 +127,34 @@ em que foram criados, no formato **YYYYMMDDHHMMSS_MigrationName.php**. Aqui est
...
 database, you will usually provide two things:
+Quando utilizar o ``bake`` para criar as migrações, você normalmente precisará informar os seguintes dados::
 
-* the name of the migration you will generate (``CreateProducts`` in our
-  example)
-* the columns of the table that will be added or removed in the migration
-  (``name:string description:text created modified`` in our example)
+* o nome da migração que você irá gerar (``CreateProducts`` por exemplo)
+* as colunas da tabela que serão adicionadas ou removidas na migração 
+(``name:string description:text created modified`` no nosso caso)
```
Comment (cake17) [inline pt/migrations.rst]: This line needs 2 spaces in the beginning

**Comment by cake17** [comment]
thanks @angelitomg ! I added a note on top of the document to explain this is not yet complete in 237e4ad5943ed5d87769b0ae963aa02e97326c3c

**Comment by cake17** [comment]
@angelitomg Maybe you can be get some inspiration from this PR #3788

**Pull request merged**